cohort-setup.md

```diff
@@ -19,10 +21,47 @@ Format is `yyyy-mm-dd`.
 
 ## 2. Source files
 
-Add a folder to the `price-migration-engine-prod` bucket with the same name as the cohort in the cohort specification
+Add a folder to the `price-migration-engine-prod` S3 bucket with the same name as the cohort in the cohort specification
 table.  
 Add two CSV files to the folder:
 * **salesforce-subscription-id-report.csv**: A file holding line-separated subscription numbers to be **included** in the 
-price rise, which will typically be the result of a Salesforce report.
+price rise, which will typically be the result of a Salesforce report. More info in [ImportSubscriptionId.MD](./ImportSubscriptionId.MD) on this. 
 * **excluded-subscription-ids.csv**: A file holding line-separated subscription numbers to be **excluded** from the price 
 rise.
+
+## 3. Run the state machine
+
+Navigate to `price-migration-engine-cohort-steps-PROD` from the Step Functions dashboard. Click on Start Execution and configure the input JSON. Look at previous executions for examples of what the input JSON should look like. E.g.: `{"cohortSpec":{"cohortName":"HomeDelivery2022","brazeCampaignName":"SV_PrintPriceRise_2022","importStartDate":"2022-04-28","earliestPriceMigrationStartDate":"2022-06-29"}}`
```

Comment:
Might be easier to run the kick-off lambda [price-migration-lambda-PROD](https://eu-west-1.console.aws.amazon.com/lambda/home?region=eu-west-1#/functions/price-migration-lambda-PROD).  
That would have the advantage of testing if the cohort spec entry is correct as well.